### PR TITLE
chore: update MIGRATING.md

### PR DESCRIPTION
**Pull request by Kosai106**
Cleanup the migration guide

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ A response resolver now exposes a single object argument instead of `(req, res,
 To mock responses, you should now return a Fetch API `Response` instance from the response resolver. You no longer need to compose a response via `res()`, and all the context utilities have also [been removed](#context-utilities).
 
 ```js
+import { http, Response } from 'msw'
```
Comment (kettanaito) [inline MIGRATING.md]: `Response` is a global class, we don't need to import it. Besides, `msw` doesn't export it, to begin with :D You can drop it.

**Review by kettanaito — CHANGES_REQUESTED** [review]
Hey, @Kosai106 👋 Thanks for making these guidelines a bit more clear. I've got only one remark regarding the `Response` global class. Otherwise looks great.

File: MIGRATING.md (hunk 1 above)
Comment (Kosai106) [reply MIGRATING.md]: Ah, fair! In that case there are some instances from before my PR which I'll drop too. :smile:

File: MIGRATING.md [hunk 2]
```diff
@@ -353,7 +361,7 @@ export const handlers = [
 When you provide an object as the `ResponseInit.headers` value, you cannot specify multiple response cookies with the same name. Instead, to support multiple response cookies, provide a `Headers` instance:
 
 ```js
-import { Headers, HttpResponse, rest } from 'msw'
+import { Headers, http, HttpResponse } from 'msw'
```
Comment (kettanaito) [inline MIGRATING.md]: While we're at it, maybe let's clean up other global classes like `Headers` too? No need to import them from MSW anymore, neither does it export them with 2.0.

**Review by kettanaito — APPROVED** [review]

**Comment by kettanaito** [comment]
Thanks, @Kosai106 👏 Welcome to contributors.

**Pull request merged**